### DISCUSSION TITLE
夸克网盘打不开文件，但是使用浏览器下载时出现以下报错

### DISCUSSION THREAD

**Discussion by Tropical8818** [AlistGo/alist#2990] [Q&A]
{"code":500,"message":"Get \"https://dl-pc-zb.drive.quark.cn/…………………………………………………………………………………………………………\": dial tcp ************02:443: i/o timeout","data":null}

说明：1. 上面的内容中省略号部分随文件不同而不同，其余部分每次显示都一样重复出现。望能尽快解决。2. 我查过所有Alist主页documentation内容亦无解决方案。

**Comment by NovaShen0724** [comment]
给个复现链接/更新下令牌

**Reply by Tropical8818** [reply]
https://alist.lucky-star.ml/Quark/龙族/00.mp4
